libs/src/api/File.ts

```diff
@@ -99,6 +101,7 @@ export class File extends Base {
                 {
                   method: 'get',
                   responseType
+                  // todo: Do we need to also set the premium content headers in this legacy flow?
```

Comment:
do we handle this flow? does it get hit and does it go to the CN /ipfs/:CID route?

Reply:
good q, and I'm not sure - maybe someone from client or content knows?

Reply:
@vicky-g maybe?

Reply:
adding the headers here too for safety